src/eval-frame/index.jsx

```diff
@@ -48,3 +47,5 @@ render(
   </Provider>,
   document.getElementById("view-mode-styles")
 );
+
+initializeUserVariables();
```

Comment:
remove `store` param, and move after render (since this is not needed immediately when the window opens)